### ISSUE TITLE
[Feature] Disable /pweather

### ISSUE THREAD

**Issue by Warriorrrr** [EarthMC/Issue-Tracker#1061]
**Is your feature request related to a problem? Please describe.**
Currently, people who have paid for Premium rank have access to /pweather. Sounds pretty innocent, but /pweather can be combined with a trident to grant infinite flight. I'm sure anyone could agree with me that such a perk is simply overpowered, especially that you need to pay to unlock it.

**Describe the solution you'd like**
That /pweather gets removed/disabled/removed. Feedback is appreciated.

**Comment by Cactusinapumpkin** [comment]
What would we replace it with?

**Comment by jkmartindale** [comment]
tbh I thought Riptide was the whole point of /pweather. It never rains on the server so there's no need to set it to clear client-side, and I don't think many people are willing to drop their FPS just to get cosmetic rain on their end. Tridents are rare and expensive, so I don't think client-side rain upsets game balance too much and makes up for the lack of elytra.

If admins decide Riptide is a game balance problem, there's the option of just disabling Riptide. I know CMI can do it, and I'm sure there are lighter weight plugins that could also work.

**Comment by Oln123** [comment]
What warriorrr is describing seems like you are paying for an advantage

**Comment by Cactusinapumpkin** [comment]
Yes this is an issue because it gives you an unfair advantage in pvp and is considered pay to win. If your lucky enough to have a trident. Although it seems kinda dumb to enable rain someone might enjoy the rain

**Comment by Cactusinapumpkin** [comment]
It might seem intentional because the server is always sunny and day time and weather commands are usually to keep it that way but I believe pweather should be removed and not riptide entirely because it wouldn’t allow u to fly continuously but u could still use it to fly short distances.

**Comment by Cactusinapumpkin** [comment]
Even though the issue was unfair pvp advantage and you can still do that but just not anywhere and anytime by toggling rain

**Comment by Cactusinapumpkin** [comment]
This is not the final say on this, we are still looking into solutions to this issue, but will need to have everyone’s say and agreement before we move forward with a solution.

**Comment by jkmartindale** [comment]
rip my email notifications lol

**Comment by Warriorrrr** [comment]
https://www.youtube.com/watch?v=YaAozos0y34 This is how it looks like from the perspective of someone with a riptide trident & pweather. I think a potential fix would be to fully remove pweather & enable weather cycle, although Fix has stated he has this disabled since he "dislikes the rain". Another fix could be to disable weather entirely again.

**Comment by Cactusinapumpkin** [comment]
If we enable rain we enable /suicide

**Comment by Cactusinapumpkin** [comment]
If we were to remove do you guys have a purely cosmetic suggestion to add in its place? Something that would make premium more enticing to pay for that isn’t “pay to win” or give unfair advantage?

**Comment by jkmartindale** [comment]
> If we were to remove do you guys have a purely cosmetic suggestion to add in its place?

idk, maybe add in another legacy donator rank benefit like /playtime

Personally I'd really like the ability to change my Premium name accent to a color that isn't pink, but I don't know how feasible that is with ChatControl Pro

**Comment by FutureLemon** [comment]
> If we were to remove do you guys have a purely cosmetic suggestion to add in its place? Something that would make premium more enticing to pay for that isn’t “pay to win” or give unfair advantage?

Trails, it's a thing the Admins have been playing with before and could finally be implemented.

**Comment by KarlOfDuty** [comment]
Should be done.

**Comment by Warriorrrr** [comment]
/pweather seems to still work in game, could you take a look at this?
